modules/installation/pages/server-setup.adoc

```diff
@@ -28,58 +28,50 @@ The {productname} Server must be managed individually.
 
 == Set up {productname} with YaST
 
-This section will guide you through {productname} setup procedures.
+This section guides you through {productname} setup using {yast}.
+
+
 
 .Procedure: {productname} Setup
 . On the {productname} Server, at the command line, use the [command]``yast2 susemanager_setup`` command to begin setup.
-
 . From the introduction screen select menu:SUSE Manager Setup[Setup SUSE Manager from scratch] and click btn:[Next] to continue.
-
 . Enter an email address to receive status notifications and click btn:[Next] to continue.
 {productname} can sometimes send a large volume of notification emails.
 You can disable email notifications in the {webui} after setup, if you need to.
-
 . Enter your certificate information and a password.
+If you intend to use a custom SSL certificate, you need to have set this up first.
+For more information about SSL certificates, see xref:administration:ssl-certs.adoc[].
+. Click btn:[Next] to continue.
+. From the menu:SUSE Manager Setup[Database Settings] screen, enter a database user and password and click btn:[Next] to continue.
 Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
 Always store your passwords in a secure location.
-+
+. Click btn:[Next] to continue.
+. Click btn:[Yes] to run setup when prompted, and wait for it to complete.
+. Click btn:[Next] to continue.
+Take a note of the address of the {productname} {webui}.
+. Click btn:[Finish] to complete {productname} setup.
+
 
 [IMPORTANT]
-.Certificate Password
 ====
+When you create your certificate password, ensure it is at least seven characters in length.
```

Comment:
If we repeat the password requirements here, we probably can remove it some 10 lines above.

Reply:
Ah, my mistake, I thought I took it out of the procedure.